### DISCUSSION TITLE
FEAT: Ability to change all margin, padding etc properties together

### DISCUSSION THREAD

**Discussion by kuhelbeher** [GrapesJS/grapesjs#3532] [Ideas]
Hello, right now grapesjs provides built in properties like margin, padding that allow user to change each property one by one (margin-top, margin-bottom, etc).

It would be nice to have ability to change all these properties together. To have some lock button, for example, when it's enabled it will change all properties together, not one by one. It could be look like this:

![image](https://user-images.githubusercontent.com/37115006/119662534-2bf83700-be3a-11eb-83ff-09b7b63446d5.png)

@artf is there an easy way to implement this using existing grapesjs tools, e.g. StyleManager.addType method or something similar?

**Comment by artf** [comment]
Yeah, that would be cool, but unfortunately, GrapesJS doesn't have yet the proper UI module which could allow easily to build new UI components by reusing others, so [`StyleManager.addType`](https://grapesjs.com/docs/api/style_manager.html#addtype) is the only way but in that case, it's up to you building all the elements.
